developer_manual/app/backgroundjobs.rst

```diff
@@ -4,14 +4,22 @@ Background Jobs (Cron)
 
 .. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
 
-Background/cron jobs are usually registered in the :file:`appinfo/app.php` by using the **addRegularTask** method, the class and the method to run:
+Background/cron jobs should be registered in the :file:`appinfo/install.php` and :file:`appinfo/update.php`. Previously this was mostly done in :file:`appinfo/app.php`, but that causes a query on each page load.
+Background jobs have to implement the ``OCP\BackgroundJob\IJob`` interface, then you can simply add them to the JobList object.
```

Comment:
See code bellow, we have some samples, but they are not OCP, maybe I will even fix that one day.